### DISCUSSION TITLE
Saving Mt. Wilson Observatory: Inside the long battle to maintain the spot where we found our place in the universe

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#164] [General]
[https://www.latimes.com/science/story/2023-10-03/saving-mt-wilson-observatory-battle-maintain-spot-found-universe](https://www.latimes.com/science/story/2023-10-03/saving-mt-wilson-observatory-battle-maintain-spot-found-universe)

**Comment by bgreenman** [comment]
Mt. Wilson should be preserved for its history and use by amateur astronomers
